sstables/sstables.cc

```diff
@@ -1802,6 +1810,7 @@ sstable::load_owner_shards(const dht::sharder& sharder) {
     if (!has_valid_sharding_metadata) {
         sstlog.warn("Sharding metadata not available for {}, so Summary will be read to allow Scylla to compute shards owning the SSTable.", get_filename());
         co_await read_summary();
+        co_await read_partitions_db_footer();
```

Comment:
Could this poison the sstable when loading it from the wrong shard?

AFAIR not, we'd convert the sstable to an foreign_sstable_open_info, ship that to the other shard, then reopen it, but I might be misremembering.

Reply:
"Poison"? What do you mean?

Reply:
Loading wrong-shard data then using it in the wrong place.

It may be echos of very old problems we had, which then prompted us to introduce foreign_sstable_open_info.